.eslintrc.js

```diff
@@ -3,5 +3,18 @@ module.exports = {
     node: true,
     es6: true,
   },
-  extends: ["eslint:recommended"],
+  extends: ["@stellar/eslint-config"],
+  rules: {
+    "import/no-extraneous-dependencies": "off",
+    "import/no-unresolved": "off",
+    "react-hooks/exhaustive-deps": "off",
```

Comment:
I think this rule is a little aggressive, since you have to annotate each variable you use in useEffect which I find verbose, but I can be convinced otherwise.

Reply:
I've been operating with the rule on, I just take it as a matter of course that I need to control references if I don't want something to re-run unnecessarily. I've definitely hit some bugs (can't remember any top of my head) that were fixed by enumerating deps + controlling references.

Reply:
Cool, then let's get rid of it.

Reply:
I kept getting errors on react-native part so copied and pasted `mobile-wallet`'s eslintrc - https://github.com/stellar/mobile-wallet/blob/master/.eslintrc.js